OpenRA.PathfinderTests/PathfinderTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿using Moq;
+using NUnit.Framework;
+using OpenRA;
+using OpenRA.Mods.Common.Traits;
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace PathfinderTests
+{
+	[TestFixture]
+	public class PathfinderTests
+	{
+		private IMap BuildFakeMap(int mapWidth, int mapHeight)
+		{
+			var map = new Mock<IMap>();
```

Comment:
What's the advantage of this over defining a minimal TestMap/TestWorld class that returns the same results?